karma.conf.js

```diff
@@ -72,8 +72,9 @@ module.exports = function(config) {
     concurrency: USING_SL ? 4 : Infinity,
     customLaunchers: customLaunchers,
 
-    coverageReporter: {
+    coverageIstanbulReporter: {
       dir: 'coverage',
+      skipFilesWithNoCoverage: true,
```

Comment:
What's the reason for adding this? Would this risk not reporting files under `packages` that are somehow not tested at all?

Reply:
docs aren't super clear here `// Omit files with no statements, no functions and no branches from the report`.

I think I can remove it, since none of the test files are empty anyways.